proto/workspace.proto

```diff
@@ -12,48 +14,57 @@ package keboola.storageDriver.command.workspace;
  * - if read only storage is supported project read only role is also assigned to workspace role so user can read from all project tables
  */
 message CreateWorkspaceCommand {
-  string projectId = 1;
-  string workspaceId = 2;
-  string projectUserName = 3;
-  string projectRoleName = 4;
-  string projectReadOnlyRoleName = 5;
+  string stackPrefix = 1; // static prefix of stack used
+  string projectId = 2; // Keboola Connection project id, id is currently numeric, but string here as this could change in the future
+  string workspaceId = 3; // Keboola Connection workspace id
```

Comment:
@zajca neni tohle blbost, kdyz ten workspace teprve vytvarim? Co by melo byt `workspaceId`?

Reply:
resp. tady uz to ID nezjistim, takze to budu muset nejak ohackovat, ne?
https://github.com/keboola/connection/blob/master/legacy-app/application/src/Storage/Workspace/Service/WorkspacesService.php#L167-L171

Reply:
jo, uz vyreseno